shared/src/main/scala/coop/rchain/sdk/casper/syntax/CasperDagDataSyntax.scala

```diff
@@ -0,0 +1,76 @@
+package coop.rchain.sdk.casper.syntax
+
+import coop.rchain.sdk.dag.data.DagData
+import coop.rchain.sdk.dag.syntax._
+
+trait CasperDagDataSyntax {
+  implicit def sdkCasperSyntaxDagDataMessage[F[_], M, S](m: M): CasperDagDataMessageOps[M, S] =
+    new CasperDagDataMessageOps(m)
+}
+
+sealed trait MalformedMessageReason
```

Comment:
It seems that `sealed` can only be expanded in the same source file. Not sure if you want to expand this in another file